back/package.json

```diff
@@ -5,12 +5,23 @@
   "description": "Email VC Issuer backend",
   "main": "dist/scripts/testing.ts",
   "scripts": {
-    "test": "npx jest",
+    "test": "npx jest --coverage",
     "build": "npx tsc",
-    "serve:test": "npm run build && node dist/scripts/testing.js"
+    "serve": "npm run build && node dist/index.js"
```

Comment:
Can we keep the testing script?

Reply:
I've renamed it because it is the script we will use to deploy the sandbox/prod standalone service as well, it will not be testing anymore

Reply:
I mean leaving both scripts 🙂